libsolidity/interface/ErrorReporter.cpp

```diff
@@ -80,6 +84,16 @@ void ErrorReporter::error(Error::Type _type, SourceLocation const& _location, Se
 	m_errorList.push_back(err);
 }
 
+void ErrorReporter::abortIfExcessive()
+{
+	if (m_errorList.size() > 256)
+	{
+		auto err = make_shared<Error>(Error::Type::Warning);
+		*err << errinfo_comment("Dude, there are way too many errors. Chill out.");
```

Comment:
I have nothing against having some fun with the messages, but I would prefer a less gendered term than "Dude".